### DISCUSSION TITLE
ISCE install by cmake

### DISCUSSION THREAD

**Discussion by ES20180816** [isce-framework/isce2#378] [Q&A] [answered]
Hello everyone,

I try to install the ISCE-2.5.3 by cmake mode and have placed the RelaxIV to the path of  'contrib/UnwrapComp/src/RelaxIV', but when run "make install", a bug still exists as the attached file：

![picture](https://user-images.githubusercontent.com/42429090/140496408-259cc23c-75bb-429f-999f-2fa55e4b33b6.png)

**Comment by bjmarfito** [comment, marked as answer]
Copy the OPTUtils.h, MCFClass.h and RelaxIV.h to contrib/UnwrapComp/include. Then, recomplie by typing `make clean` and afterwards `make install` on the terminal.

**Reply by ES20180816** [reply]
> Copy the OPTUtils.h, MCFClass.h and RelaxIV.h to contrib/UnwrapComp/include. Then, recomplie by typing `make clean` and afterwards `make install` on the terminal.

Dear bjmarfito,

According to your suggestion, this problem has been solved correctly. 
Thank you very much.
